packages/ui/src/components/filters/filter-box-wrapper.tsx

```diff
@@ -73,11 +72,7 @@ const FilterBoxWrapper = ({
             </DropdownMenu.Trigger>
             <DropdownMenu.Content align="start">
               <DropdownMenu.Item onSelect={() => handleRemoveFilter()} asChild>
-                <Button
-                  size="xs"
-                  variant="custom"
-                  className="data-[highlighted]:bg-cn-background-transparent gap-x-1.5 data-[highlighted]:text-cn-foreground-danger"
-                >
+                <Button size="sm" variant="ghost" className="data-[highlighted]:text-cn-foreground-danger">
```

Comment:
are we using data-[highlighted] as a pseudo-selector to change the color on tab-highlight?
if so, why is not the standard behaviour for a danger button? why do we need to apply this class manually here?

Reply:
This is the behavior mentioned in figma. If we apply danger button, it will be always red color. Hence added `data-[highlighted]` for hover. Please check the figma references below:

Without hover:
<img width="311" alt="image" src="https://github.com/user-attachments/assets/4a1c6a4b-8f0e-4025-a050-03f5d5161be8" />
---
With hover: 
<img width="355" alt="image" src="https://github.com/user-attachments/assets/26b0853c-a733-4731-acc1-322dfa0c076f" />